src/app/DataVersionFilter.h

```diff
@@ -35,6 +36,17 @@ struct DataVersionFilter
         return (mEndpointId != kInvalidEndpointId) && (mClusterId != kInvalidClusterId) && (mDataVersion.HasValue());
     }
 
+    bool operator==(const DataVersionFilter & aOther) const
+    {
+        return mEndpointId == aOther.mEndpointId && mClusterId == aOther.mClusterId &&
+            mDataVersion.Value() == aOther.mDataVersion.Value();
+    }
+
+    bool operator<(const DataVersionFilter & aOther) const
```

Comment:
Where do we use this?

Reply:
good catch, in latest code, i am not using that, removed ,thanks